NightscoutServiceKit/OTPManager.swift

```diff
@@ -70,7 +70,8 @@ public class OTPManager {
     
     private var secretStore: OTPSecretStore
     private var nowDateSource: () -> Date
```

Comment:
The fix: Changing Sha512 -> Sha51
Also adding up the "Issuer" as a property as a I reuse it.